javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
Okay, I went this way but I'm a bit stuck on how to serialize the CdoSnapshotState including the new version field. Currently it is serialized as `state: {prop1: val1, ..}`. I can either add the version field inside of the state as another property (but then it could already have a property with the same name which means a conflict) or serialize it as `state: {version: X, properties:  {prop1: val1, ..}}`, but this wouldn't be backward compatible. Any ideas on this?